### ISSUE TITLE
should roq.java not do actual roq things?

### ISSUE THREAD

**Issue by maxandersen** [quarkiverse/quarkus-roq#133]
roq.java seem to be just serving content, like what `jbang httpd@jbangdev -d target/roq -p 8181` does for Java 8+
or since Java 18 there is `jwebserver -d target/roq -b 8181`

shouldn't we make `roq.java` do actual roq stuff?

**Comment by ia3andy** [comment]
yeah I was thinking about adding the ability to generate but I am not sure how we could do it.
